test/test_unified_format.py

```diff
@@ -36,15 +36,11 @@
     ]))
 
 
-globals().update(generate_test_classes(
-    os.path.join(_TEST_PATH, 'valid-fail'),
-    module=__name__,
-    class_name_prefix='UnifiedTestFormat',
-    expected_failures=[
-        'foo',
-        'FindOneAndReplace returnDocument invalid enum value',
-        'FindOneAndUpdate returnDocument invalid enum value'
-    ]))
+# Uncomment to run the valid-fail tests
+# globals().update(generate_test_classes(
+#     os.path.join(_TEST_PATH, 'valid-fail'),
+#     module=__name__,
+#     class_name_prefix='UnifiedTestFormat'))
```

Comment:
Perhaps we can uncomment this by adding a flag `expect_failure=True`. When `expect_failure=True` we can ignore errors that happen at test construction time, if the test actually does get constructed then we should assert that test fails at runtime. What do you think?

Reply:
That is too similar to `expected_failures`. How about a `bypass_test_generation_errors` kwarg instead that allows us to ignore/bypass errors raised during test creation?